pom.xml

```diff
@@ -124,7 +124,7 @@
     <failsafe.excludedGroups />
     <failsafe.forkCount>1</failsafe.forkCount>
     <failsafe.groups />
-    <hadoop.version>3.2.1</hadoop.version>
+    <hadoop.version>3.3.0</hadoop.version>
```

Comment:
I'd be okay with bumping to 3.2.2, because I think it's reasonable to support the latest bugfix in the earlier Hadoop release line. If we bump to 3.3.0, we may risk making changes that work for 3.3.x, but will cause a failure with 3.2.x. However, that may also be okay, since we have a build test to ensure it builds with versions as old as 3.0.3. What do you think?

Reply:
We can go with 3.2.2 just to take a more cautious approach. No need to chance breaking something if not necessary. I'll make the change.

Reply:
I don't even see 3.2.2 listed on their website. Maybe it would be best to stick with 3.3.0 as the default.

Reply:
Interesting! I'll revert back to 3.3.0 :)